python/paddle/distributed/fleet/data_generator/data_generator.py

```diff
@@ -156,13 +156,13 @@ def generate_sample(self, line):
 
         Returns:
             Returns the data processed by the user.
-              The data format is list or tuple:
-            [(name, [feasign, ...]), ...]
-              or ((name, [feasign, ...]), ...)
+                The data format is list or tuple:
+                [(name, [feasign, ...]), ...]
+                or ((name, [feasign, ...]), ...)
 
             For example:
```

Comment:
For example: 
            [("words", [1926, 08, 17]), ("label", [1])]
            or (("words", [1926, 08, 17]), ("label", [1]))